### ISSUE TITLE
gtksourceview background color not working properly

### ISSUE THREAD

**Issue by RalfJung** [dirruk1/gnome-breeze#69]
The GTK2-based application CoqIDE uses gtksourceview to display source code, and it uses background color (per statement) to display the state of the statement. Background colors are not displayed correctly with the breeze theme.

Here's a screenshot of CoqIDE with breeze: <https://coq.inria.fr/bugs/attachment.cgi?id=805>
And here's a screenshot with Clearlooks:
![coqide2](https://cloud.githubusercontent.com/assets/330628/21989231/6ba69728-dc0a-11e6-8553-645766bd245f.png)

CoqIDE 8.5 is packaged in Debian and probably other distributions. The screenshots above are with CoqIDE 8.6, which is available via opam ("opam install coqide"), where opam in turn is oackaged in Debian and probably other distributions.

**Comment by dirruk1** [comment]
I could have a look at it but I'm not sure what the expected behaviour is. This is not about text selection right? What exactly do you mean by state of the statement?

**Comment by RalfJung** [comment]
No, it's not about text selection. Coq is some sort of an interpreter that interprets files statement by statement. One essentially steps through the program one statement ("."-separated "thing") after the other. The IDE uses the background color to indicate how far ahead one is through the program.

coqide is e.g. packaged for Debian. If you want to test this yourself, you can create a file `test.v` with content
```
Definition foo := True.
```
then open it in CoqIDE and run "Navigation - End". With the breeze theme, the background remains a light yellow. With the clearlooks theme, the background behind the text up until the "." will turn green.

I don't know the interface to gtksourceview, so I can't tell what CoqIDE is doing to achieve this effect.

EDIT:
Just comparing these two images should also clarify what is missing:
<https://coq.inria.fr/bugfiles/attachment.cgi?id=805>
<https://cloud.githubusercontent.com/assets/330628/21989231/6ba69728-dc0a-11e6-8553-645766bd245f.png>
